### ISSUE TITLE
Unrecognized oncogenes

### ISSUE THREAD

**Issue by hansenp** [TheJacksonLaboratory/VPV#125]
The three gene symbols:

DUX4
KIAA1524
UGT1A

 of the cancer gene list:

https://github.com/TheJacksonLaboratory/VPV/blob/master/src/test/resources/oncogenes.txt

Cannot be found in RefSeq annotation. Will look for alternative symbols...

**Comment by hansenp** [comment]
Found alternative gene symbol KMT2A for KIAA1524 in COSMIC's cancer gene census table that is also in RefSeq.

**Comment by hansenp** [comment]
The gene UGT1A has 10 entries in refGene.txt:
```
296	NM_205862	chr2	+	234600320	234681945	234602451	234681205	6	234600320,234602444,234675679,234676494,234676865,234680907,	234600466,234602511,234675811,234676582,234677085,234681945,	0	UGT1A6	cmpl	cmpl	-1,0,0,0,1,2,
296	NM_019076	chr2	+	234526290	234681945	234526353	234681205	5	234526290,234675679,234676494,234676865,234680907,	234527208,234675811,234676582,234677085,234681945,	0	UGT1A8	cmpl	cmpl	0,0,0,1,2,
296	NM_019075	chr2	+	234545122	234681951	234545168	234681205	5	234545122,234675679,234676494,234676865,234680907,	234546023,234675811,234676582,234677085,234681951,	0	UGT1A10	cmpl	cmpl	0,0,0,1,2,
296	NM_001072	chr2	+	234601511	234681951	234601650	234681205	5	234601511,234675679,234676494,234676865,234680907,	234602511,234675811,234676582,234677085,234681951,	0	UGT1A6	cmpl	cmpl	0,0,0,1,2,
296	NM_021027	chr2	+	234580543	234681951	234580580	234681205	5	234580543,234675679,234676494,234676865,234680907,	234581435,234675811,234676582,234677085,234681951,	0	UGT1A9	cmpl	cmpl	0,0,0,1,2,
2375	NM_007120	chr2	+	234627437	234681945	234627466	234681205	5	234627437,234675679,234676494,234676865,234680907,	234628333,234675811,234676582,234677085,234681945,	0	UGT1A4	cmpl	cmpl	0,0,0,1,2,
2375	NM_019078	chr2	+	234621637	234681945	234621637	234681205	5	234621637,234675679,234676494,234676865,234680907,	234622504,234675811,234676582,234677085,234681945,	0	UGT1A5	cmpl	cmpl	0,0,0,1,2,
296	NM_019077	chr2	+	234590583	234681945	234590583	234681205	5	234590583,234675679,234676494,234676865,234680907,	234591438,234675811,234676582,234677085,234681945,	0	UGT1A7	cmpl	cmpl	0,0,0,1,2,
2375	NM_019093	chr2	+	234637772	234681945	234637772	234681205	5	234637772,234675679,234676494,234676865,234680907,	234638639,234675811,234676582,234677085,234681945,	0	UGT1A3	cmpl	cmpl	0,0,0,1,2,
2375	NM_000463	chr2	+	234668918	234681945	234668933	234681205	5	234668918,234675679,234676494,234676865,234680907,	234669797,234675811,234676582,234677085,234681945,	0	UGT1A1	cmpl	cmpl	0,0,0,1,2,
```
The locus encodes for at least 9 genes due to alternative splicing. See here:

https://www.ncbi.nlm.nih.gov/gene?Db=gene&Cmd=ShowDetailView&TermToSearch=7361
https://en.wikipedia.org/wiki/UGT1A@

@pnrobinson: Which is relevant for cancer? There is no entry in COSMIC's cancer gene census table.

 Maybe we should take them all, i.e. UGT1A1, UGT1A3, UGT1A4, ..., UGT1A10.

**Comment by hansenp** [comment]
DUX4 is in refGene.txt but it occurs only on a non-canonical chromosome:
```
585	NM_001293798	chrUn_gl000228	+	112604	114676	112604	113879	3	112604,114023,114477,	113887,114115,114676,	0	DUX4	cmpl	cmpl	0,-1,-1,
585	NM_001306068	chrUn_gl000228	+	112604	114676	112604	113879	2	112604,114477,	114115,114676,	0	DUX4	cmpl	cmpl	0,-1,
585	NR_137167	chrUn_gl000228	+	112604	124171	124171	124171	6	112604,114023,121008,121493,123599,124108,	113887,114115,121073,121561,123698,124171,	0	DUX4	unk	unk-1,-1,-1,-1,-1,-1
```

**Comment by hansenp** [comment]
In hg38 DUX4 occurs on chr4.

**Comment by pnrobinson** [comment]
Yes we should probably do this on HG38 to get DUX4. I would say that for now there is no need to take the UGT1A1 family, at least to nmy knowledge these genes are important for pharmacogenomics.

**Comment by hansenp** [comment]
Both KIAA1524 by KMT2A are in `oncogenes.txt`. I made mistake. KIAA1524 is not a synonym for KMT2A, it is (according to COSMIC) a translocation partner of KMT2A.

NCBI gene says the official symbol for KIAA1524 is CIP2A (https://www.ncbi.nlm.nih.gov/gene/57650).

We could just delete KIAA1524 from the list, or we could replace it by CIP2A. What do you prefer?

**Comment by pnrobinson** [comment]
I updated the oncogenes file to change KIAA1524 to CIP2A. Using the hg38 genome (note -- I needed to change some code for the new g-unzipper, so everything needs to be done afresh), everything is recognized, except for UGT1A. I now understand that the genes are actually UGT1A1, UGT1A2, etc., and that there is no gene called simply UGT1A. I will remove UGT1A. UGT1A1 is already in the list, which is appropriate since it is associated with some cancer types.  I think we are no good to go with this list.
